symbolic-symcache/src/lookup.rs

```diff
@@ -107,49 +103,36 @@ impl<'data> SymCache<'data> {
 ///   - path_name: pthread.h
 #[derive(Debug, Clone)]
 pub struct File<'data> {
-    /// The optional compilation directory prefix.
-    pub comp_dir: Option<&'data str>,
     /// The optional directory prefix.
     pub directory: Option<&'data str>,
     /// The file path.
-    pub path_name: &'data str,
+    pub name: &'data str,
```

Comment:
I can’t remember why these were `pub`. Are they globally pub, or just inside the crate? In which case, we should make them private, as we have accessors for them anyway